provisioner/roles/manage_ec2_instances/defaults/main.yml

```diff
@@ -70,3 +69,19 @@ ec2_info:
     architecture: x86_64
     filter: 'F5 Networks Prelicensed Hourly BIGIP-13.1*Good 25MBPS*'
     username: admin
+  splunk_enterprise:
+    owners: 309956199498
+    size: c4.4xlarge
+    os_type: linux
+    disk_space: 10
```

Comment:
Please increase this. With 10GB we run into warnings directly after deployment, and run out of storage quickly as soon as the add-on is installed. To be save I'd suggest something around 25 GB, or even 50GB?

Reply:
I will defer to Roland, we are not getting beat up yet by pricing models, etc so I would encourage you to just increase it, so its reliable